projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/experiment/Experiment.java

```diff
@@ -159,6 +159,15 @@ public void addSpecimens(Collection<Specimen> specimens) {
     this.specimens.addAll(missingSpecimens);
   }
 
+  public void removeAllExperimentalVariables() {
```

Comment:
JavaDoc detailing that this will also affect experimental groups would be good here.